packages/airnode-deployer/config/config.json.example

```diff
@@ -33,11 +33,12 @@
     },
     "httpGateway": {
       "enabled": true,
-      "apiKey": "${HTTP_GATEWAY_API_KEY}"
+      "apiKey": "${HTTP_GATEWAY_API_KEY}",
+      "maxConcurrency": 20
     },
     "logFormat": "plain",
     "logLevel": "INFO",
-    "nodeVersion": "0.4.0",
+    "nodeVersion": "0.5.0",
```

Comment:
Shouldn't this be changed for all `nodeVersion` we have in the project at the begginning of v0.5 (and validator templates copied)?

Reply:
EDIT: I see you are doing this in this PR, but shouldn't also these change:
- https://github.com/api3dao/airnode/blob/concurrency-reservation-gateway/packages/airnode-validator/exampleSpecs/config.specs.json#L32
- https://github.com/api3dao/airnode/blob/concurrency-reservation-gateway/packages/airnode-validator/exampleSpecs/secrets.env#L4
- https://github.com/api3dao/airnode/blob/concurrency-reservation-gateway/packages/airnode-validator/src/validator.test.ts#L400

Reply:
Yeah, I guess so. TBH updating `config.json` is such a painful process. We should do something about that.

Reply:
Why is it painful? Isn't it just find and replace? (with some copying)?

Reply:
Depending on what you're changing in the `config.json` it can go through multiple packages, source files, and tests. And as you can see I still missed a few :smile: It was actually worse in https://github.com/api3dao/airnode/pull/820/commits/bdeb56a9eb47e23724295dd2289bbb4b3e86b70c.